### PR TITLE
D8CORE-2610 Deprecation notice about C, G, J, and Q lockup

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Deprecation notice about C, G, J, and Q lockup

# Needed By (Date)
- Within V6

# Urgency
- Non-urgent

# Steps to Test

# Associated Issues and/or People
-  [D8CORE-2610 ](https://stanfordits.atlassian.net/browse/D8CORE-2610)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by sherakama** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr788-blt7lamun86zd3p995zmfttjehdq2ard.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ffe1e31dc54c3b29f9b3fc4

File: DEPRECATIONS.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Deprecation Notices
 
-*Deprecated in 6.2.0*
+_Deprecated in 6.2.0_
+
 - `modular-spacing` mixin will be removed in Version 7. Please use `responsive-spacing` instead.
 - Noto Sans Font will be removed in Version 7
 - Kalam Font will be removed in Version 7.
 - `su-brand-bar` will be removed in Version 7, and replaced with `su-global-identity-bar`.
+- Stanford University lockup options C, G, J, and Q (i.e. `.su-lockup--option-c`) will be removed in Version 7.
```
Comment (sherakama) [inline DEPRECATIONS.md:9]: This notice should be moved to a new heading `Deprecated in 6.2.2` as 6.2.1 has already been released.

Also, I do not see a deprecation comment in the SCSS eg: https://github.com/SU-SWS/decanter/blob/master/core/src/scss/components/lockup/_lockup--option-c.scss

Please add deprecation notice SCSS comment to each of the lockups. EG: https://github.com/SU-SWS/decanter/blob/master/core/src/scss/core/fonts/_fonts.scss#L21 and https://github.com/SU-SWS/decanter/blob/master/core/src/scss/utilities/mixins/modular-scale/_modular-spacing.scss#L22

**Review by sherakama — CHANGES_REQUESTED** [review]
Some feedback.

**Comment by sherakama** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr788-blt7lamun86zd3p995zmfttjehdq2ard.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ffe1e31dc54c3b29f9b3fc4

File: DEPRECATIONS.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # Deprecation Notices
 
-*Deprecated in 6.2.0*
+_Deprecated in 6.2.2_
```
Comment (sherakama) [inline DEPRECATIONS.md:3]:
```suggestion
_Deprecated in 6.2.2_

- Stanford University lockup options C, G, J, and Q (i.e. `.su-lockup--option-c`) will be removed in Version 7.

_Deprecated in 6.2.0_
```

File: DEPRECATIONS.md [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+
 - `modular-spacing` mixin will be removed in Version 7. Please use `responsive-spacing` instead.
 - Noto Sans Font will be removed in Version 7
 - Kalam Font will be removed in Version 7.
 - `su-brand-bar` will be removed in Version 7, and replaced with `su-global-identity-bar`.
+- Stanford University lockup options C, G, J, and Q (i.e. `.su-lockup--option-c`) will be removed in Version 7.
```
Comment (sherakama) [inline DEPRECATIONS.md:9]:
```suggestion
```

File: core/src/scss/components/lockup/_lockup.scss [hunk 4]
```diff
@@ -7,21 +7,21 @@
...
 // .su-lockup--option-l - Option L
 // .su-lockup--option-m - Option M
 // .su-lockup--option-n - Option N
 // .su-lockup--option-o - Option O
 // .su-lockup--option-p - Option P
-// .su-lockup--option-q - Option Q
+// .su-lockup--option-q - Option Q will be removed in Version 7
 // .su-lockup--option-r - Option R
 // .su-lockup--option-s - Option S
 // .su-lockup--option-t - Option T
```
Comment (sherakama) [inline core/src/scss/components/lockup/_lockup.scss:27]:
```suggestion
// .su-lockup--option-t - Option T

// @deprecated Options C, G, J, & Q will be removed in Version 7

```

**Comment by rebeccahongsf** [comment]
Thank you @sherakama for the speedy review and feedback! 😄  

Would you happen to know if the information listed on our [Decanter Lockup page ](https://decanter.stanford.edu/component/identity-lockup/) needs to be deprecated through Netlify/Decanter-web or within this repo? If it's within this repo, I'd like to include it with this PR.

**Comment by sherakama** [comment]
> Thank you @sherakama for the speedy review and feedback! 😄
> 
> Would you happen to know if the information listed on our [Decanter Lockup page ](https://decanter.stanford.edu/component/identity-lockup/) needs to be deprecated through Netlify/Decanter-web or within this repo? If it's within this repo, I'd like to include it with this PR.

I suppose the answer is both. The code itself should notify developers about deprecations. I cannot remember if we had direct handling on deprecations from SCSS comments but I doubt it. Especially from variants. I would say that the notice would have to be added to the component in the Netlify CMS https://decanter.stanford.edu/admin/index.html#/collections/components/entries/identity-lockup

**Comment by sherakama** [comment]
@rebeccahongsf it looks like you started from the forms branch. This will need to go into master so you will have to revise your branch and either rebase out the form work or copy over the work into a new branch stemmed off of master.

**Comment by rebeccahongsf** [comment]
> @rebeccahongsf it looks like you started from the forms branch. This will need to go into master so you will have to revise your branch and either rebase out the form work or copy over the work into a new branch stemmed off of master.

Thanks Shea! I'll move the work over to a new branch off of master and get a PR up tomorrow! 👍

**Pull request closed without merging**